Pipelines/Templates/export-Solution.yml

```diff
@@ -339,6 +339,19 @@ steps:
     configurationData: '${{parameters.configurationData}}' 
     buildSourceDirectory: '$(Build.SourcesDirectory)'
 
+# TEMPORARY until Solution export supports not exporting custom connector configurations
+# We need to clear out the contents of any *_connectionparameters.json file.  This is to ensure those valuse
+# are not overwritten when the Solution is imported to a downstream environment.
+- powershell: |  
```

Comment:
@RajeevPentyala definitely don't want this here. Can you move this to the Business Approval Kit export pipeline instead e.g. make the update after the export solution template is run and then do another commit?

Reply:
Is there anyway to do this in a generic way that would fix the issue we are trying to fix for the Business Approval Kit? e.g. only clear the connection parameters for oAuth connections?